homeassistant/components/airthings_ble/config_flow.py

```diff
@@ -128,15 +128,15 @@ async def async_step_bluetooth_confirm(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Confirm discovery."""
+        assert self._discovered_device is not None
```

Comment:
Is it okey to do this? This should really not be `None` in this step. Not sure how to rewrite it without this.

I previously had a `if self._discovered_device is None`, but wasn't sure how to actually test it.